Making readme more readable & typo-free (#14)
serokell/crossref-verifier: README.md

[![Build status](https://badge.buildkite.com/75461331a6058b334383cdfca1071dc1f908b70cf069d857b7.svg?branch=master)](https://buildkite.com/serokell/crossref-verifier)

> Crossref-verifier is a tool for verifying local and external references in repository documentation that is quick, easy to setup, and can be added to CI.

Reviewer: ```suggestion
Crossref-verifier is a tool for verifying local and external references in repository documentation that is quick, easy to setup, and suits for being added to CI.
```

<img src="https://user-images.githubusercontent.com/5394217/70820564-06b06e00-1dea-11ea-9680-27f661ca2a58.png" alt="Output sample" width="600"/>

### Motivation

As the project evolves, links in documentation have a tendency to get broken. This is usually because of: 
1. File movements;
2. Markdown header renames;
3. Outer sites ceasing their existence.

This tool will help you to keep references in order.

> ### Advantages

Reviewer: I'm a bit scared of calling these advantages because one can argue whether some of points below are really definitely good. E.g. some people seemed to intentionally make their tools require precise configuring because they were thinking that any sane default behaviour would not suit their common use cases. Also, I can't say that I perfectly reached all of these points and some of the solutions below may surpass our tool in separate aspects.

Author: Yeah, it could go either way, I usually try to be mistaken on the side of boldness, but I see the point: reverted it back.
